exercises/bowling/canonical-data.json

```diff
@@ -5,13 +5,21 @@
     "Score should return the game's final score, when possible",
     "For brevity the tests display all the rolls in an array;",
     "each element of the rolls array should be passed to the roll method",
-    "The final tests define situations where the score can not be returned",
-    "The expection for these tests is -1, indicating an error",
-    "In these cases you should expect an error as is idiomatic for your language",
-    "When to error is also left up to your implementation. There are two options",
-    "  - Error as soon as an invalid roll is made",
-    "  - Error when scoring a game with an invalid roll",
-    "You can also error in both cases."
+    "",
+    "If the `roll_should_fail` key is present",
+    "Then after rolling the values in the `roll` key,",
+    "attempting to roll the value in the `roll_should_fail` key should produce an error.",
+    "",
+    "If the `expected` key is present and non-negative",
+    "all rolls should succeed, and the final score should equal that value.",
+    "",
+    "If the `expected` key is present and negative,",
```

Comment:
This is in violation of #401 and #551. I would like to suggest the `error`s from #240 be used.